versions/v0.9.1/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:4
```

Comment:
Actually, I realize why I used `/tmp` now. Files that are in `/opt` should be reserved for software and add-on packages that are not part of the default installation. Since you need to run `pulp init` to setup a seed project in order to run `psci` for later versions of Purescript, it makes no sense to keep it in `/opt`, which is why I put that in `/tmp`. 

I think it's still good to have `/tmp`, but maybe avoid setting it as the working directory so it's not the default.